dxchange/exchange.py

```diff
@@ -157,7 +158,8 @@ def read_als_832(fname, ind_tomo=None, normalized=False):
         ind_dark = list(range(0, ndark))
 
     # Read image data from tiff stack.
-    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4)
+    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4,
+                                    slc=(sino, None))
```

Comment:
@dgursoy fix seems easy, just swap the loop indexing at

[`def _shape_after_slice(shape, slc):`](https://github.com/data-exchange/dxchange/blob/c057d2599b46f06986e157024193d962643411e8/dxchange/reader.py#L555l)

but will make it not back compatible. Suggestions?